### PR TITLE
docs: add snippet for `base` option

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

Closes https://github.com/withastro/astro/issues/6335

## Testing

N/A

## Docs

/cc @withastro/maintainers-docs for feedback!

### CODE REVIEW

File: packages/integrations/node/README.md [hunk 1]
```diff
@@ -98,6 +98,19 @@ app.use(ssrHandler);
 app.listen(8080);
 ```
 
+If you have a `base` specified in your `astro.config.mjs`:
```
[the remark this answers is not in the document]
Comment (ematipico) [reply packages/integrations/node/README.md]: I went for the shortest version :)

**Review by bluwy — APPROVED** [review]
Looks good to me. It would be nice to have the same change for the fastify example, but I'm not sure how to 😅 Also good to have docs pass before merging this if they have thoughts on this.

**Review by matthewp — APPROVED** [review]

**Review by Yan-Thomas — COMMENTED** [review]
Thanks @ematipico, I added one comment for pondering, overall LGTM!

File: packages/integrations/node/README.md [hunk 2]
```diff
@@ -92,7 +92,8 @@ import express from 'express';
 import { handler as ssrHandler } from './dist/server/entry.mjs';
 
 const app = express();
-app.use(express.static('dist/client/'))
+const base = "/";
```
[the remark this answers is not in the document]
Comment (ematipico) [reply packages/integrations/node/README.md:97]: Done! @Yan-Thomas , could you please review?

**Review by Yan-Thomas — APPROVED** [review]
Perfect, LGTM!

**Pull request merged**